api/interpolated_rate.md

```diff
@@ -0,0 +1,83 @@
+---
+api_name: interpolated_rate()
```

Comment:
Same as interpolated_delta, we should fix up the entire counter_agg API at some point, but this is just following the status quo.  I think the issue should go away when Eric finishes his refactoring of this code so that both `gauge_agg` and `counter_agg` return a `MetricSummary` object, which all of these APIs will work on.